### ISSUE TITLE
setting to automatically opt-in to installs

### ISSUE THREAD

**Issue by joprice** [OCamlPro/drom#152]
Since drom uses a local switch that acts as a sandbox, could the run and test commands be configured in the drom.toml to automatically apply `-y`? This prevents one from always having to remember to add the `-y` flag, especially when it in some cases is needed twice in succession, and would match cargo's behavior.

**Comment by lefessan** [comment]
I am not sure I understand, you mean:
* Add an option in the `drom.toml` file to set (or unset) the `-y` parameter
* This `-y` parameter would be given to `opam`  for all commands in the `drom test` (and `drom run`) ?

Also, I am not sure whether it should be a project option (in `drom.toml`), a user option (in `~/.config/drom/config`) or a project user option (in a new file `.drom.config`), given that different users may have different preferences (for example, I mostly always use the same switch in all my projects using `drom build --switch XXX`).

**Comment by joprice** [comment]
I agree it's not obvious where it should be configured. It does seem like a per-user preference. Adding it to the project when combined with someone attempting to use a global switch might be surprising.

**Comment by lefessan** [comment]
I modified the behavior so that, if `drom` detects that the switch is a local one (i.e. a directory, not a link), it adds -y to opam. This is implemented in #163

**Comment by lefessan** [comment]
Maybe this behavior should be disabled using a local user flag...

**Comment by joprice** [comment]
You mean like a field to opt out in the drom.toml?

**Comment by lefessan** [comment]
No, either in `~/.config/drom/config`, or in `.drom.config` in the project directory

**Comment by joprice** [comment]
I tried out the changes but still see some calls missing the argument such as `Calling opam install --deps-only ./_drom/new.opam`. Looks like the same overriding of the `y` param needs to be here:  https://github.com/OCamlPro/drom/blob/master/src/drom_lib/build.ml#L344.
